extensions.html

```diff
@@ -182,19 +184,146 @@ <h2>
         <dfn>GamepadHapticActuator</dfn> Interface
       </h2>
       <p>
-        Each <code>GamepadHapticActuator</code> corresponds to a motor or other
-        actuator that can apply a force for the purposes of haptic feedback.
+        A {{GamepadHapticActuator}} corresponds to a configuration of motors or
+        other actuators that can apply a force for the purposes of haptic
+        feedback.
       </p>
       <pre class="idl">
         [Exposed=Window]
         interface GamepadHapticActuator {
           readonly attribute GamepadHapticActuatorType type;
+          boolean canPlayEffectType(GamepadHapticEffectType type);
```

Comment:
Maybe just? 

```suggestion
          boolean canPlay(GamepadHapticEffectType type);
```

Reply:
Actually, I think `.supports()` might be better. Consider, just because gamepad supports an effect type, doesn't mean it can play it. 

```suggestion
          boolean supports(GamepadHapticEffectType type);
```

Reply:
To future-proof this, this should take a `DOMString`, not a enum value. Otherwise, this will throw.

The logic here being that you should be able to do:

```JS
const myGamesEffects = ["rumble", "bumble", "trigger-me-fumble", "dual-rumble", "pulse"]; 
// This shouldn't throw... "trigger-me-fumble" is supported in FutureBrowser 2000!
const supportedEffects = myGamesEffects.filter(effect => gamepad.actuator.vibrationActuator.canPlay(effect));
```

Alternatively, why not just ask the `vibrationActuator` to return what it supports via a frozen array? 
Then there is no need to keep asking the gamepad what it can do. 

```JS
gamepad.actuator.vibrationActuator.effects.includes("dual-rumble")
```

Reply:
Either `DOMString` or returning a `FrozenArray` sound good to me. I slightly prefer `gamepad.vibrationActuator.effects` because it makes it possible to automatically discover new effect types.